workflow-examples/src/main/java/com/redhat/parodos/examples/vmonboarding/VmOnboardingWorkFlowConfiguration.java

```diff
@@ -0,0 +1,108 @@
+			@Value("${MAIL_SERVICE_SITE_NAME_ESCALATION:site}") String mailServiceSiteName) {
+		return new ServiceNowTicketFulfillmentEscalationWorkFlowTask(mailServiceUrl, mailServiceSiteName);
+	}
+
+	@Bean(name = "serviceNowTicketFulfillmentEscalationWorkFlow")
+	@Escalation
+	WorkFlow serviceNowTicketFulfillmentEscalationWorkFlow(
+			@Qualifier("serviceNowTicketFulfillmentEscalationWorkFlowTask") ServiceNowTicketFulfillmentEscalationWorkFlowTask serviceNowTicketFulfillmentEscalationWorkFlowTask) {
+		return SequentialFlow.Builder.aNewSequentialFlow().named("serviceNowTicketFulfillmentEscalationWorkFlow")
+				.execute(serviceNowTicketFulfillmentEscalationWorkFlowTask).build();
+	}
+
+	@Bean(name = "ipAddressProvisioningWorkFlowCheckerTask")
+	IpAddressProvisioningWorkFlowCheckerTask ipAddressProvisioningWorkFlowCheckerTask(
+			@Qualifier("serviceNowTicketFulfillmentEscalationWorkFlow") WorkFlow serviceNowTicketFulfillmentEscalationWorkFlow) {
+		return new IpAddressProvisioningWorkFlowCheckerTask(serviceNowTicketFulfillmentEscalationWorkFlow,
+				new Date().getTime() / 1000 + 30);
+	}
+
+	@Bean(name = "ipAddressProvisioningWorkFlowChecker")
+	@Checker(cronExpression = "*/5 * * * * ?")
+	WorkFlow ipAddressProvisioningWorkFlowChecker(
+			@Qualifier("ipAddressProvisioningWorkFlowCheckerTask") IpAddressProvisioningWorkFlowCheckerTask ipAddressProvisioningWorkFlowCheckerTask) {
+		return SequentialFlow.Builder.aNewSequentialFlow().named("ipAddressProvisioningWorkFlowChecker")
+				.execute(ipAddressProvisioningWorkFlowCheckerTask).build();
+	}
+
+	@Bean(name = "ansibleCompletionWorkFlowCheckerTask")
+	AnsibleCompletionWorkFlowCheckerTask ansibleCompletionWorkFlowCheckerTask() {
+		return new AnsibleCompletionWorkFlowCheckerTask();
+	}
+
+	@Bean(name = "ansibleCompletionWorkFlowChecker")
+	@Checker(cronExpression = "*/10 * * * * ?")
+	WorkFlow ansibleCompletionWorkFlowChecker(
+			@Qualifier("ansibleCompletionWorkFlowCheckerTask") AnsibleCompletionWorkFlowCheckerTask ansibleCompletionWorkFlowCheckerTask) {
+		return SequentialFlow.Builder.aNewSequentialFlow().named("ansibleCompletionWorkFlowChecker")
+				.execute(ansibleCompletionWorkFlowCheckerTask).build();
+	}
+
+	@Bean(name = "serviceNowTicketCreationWorkFlowTask")
+	ServiceNowTicketCreationWorkFlowTask serviceNowTicketCreationWorkFlowTask(
+			@Qualifier("ipAddressProvisioningWorkFlowChecker") WorkFlow ipAddressProvisioningWorkFlowChecker,
+			@Qualifier("ansibleCompletionWorkFlowChecker") WorkFlow ansibleCompletionWorkFlowChecker) {
+		ServiceNowTicketCreationWorkFlowTask serviceNowTicketCreationWorkFlowTask = new ServiceNowTicketCreationWorkFlowTask();
+		serviceNowTicketCreationWorkFlowTask
+				.setWorkFlowCheckers(List.of(ipAddressProvisioningWorkFlowChecker, ansibleCompletionWorkFlowChecker));
+		return serviceNowTicketCreationWorkFlowTask;
+	}
+
+	@Bean(name = "notificationWorkFlowTask")
+	NotificationWorkFlowTask notificationWorkFlowTask(
```

Comment:
Why did you create new `NotificationWorkFlowTask`, there is existing one in PreBuilt which can be used

Reply:
Good catch @nirarg! I'll check it out and re-use. Thanks